### DISCUSSION TITLE
CONUS404: daily mean air temperature errors

### DISCUSSION THREAD

**Discussion by zcjohnson3** [hytest-org/hytest#397] [Dataset Questions]
Hello,
I received CONUS404 "skin" daily air temperature data for ~2K stream temperature sites from Amelia Snyder ([link](https://hytest-org.github.io/hytest/dataset_access/conus404_point_selection.html)) to use as part of the IWAAs Trends & Drivers project. While exploring those data, I noticed some erroneous values at a few locations. I suspect there are more locations with this error, but I have not done an exhaustive check of all ~2K of my sites. Has someone checked mean daily values from CONUS404 against values from PRISM or Daymet? Or is this a known issue? Below is a summary of a message exchange I had with Amelia regarding this issue over the past ~day. Also, please send any emails to: zjohnson@usgs.gov (not the email associated with my GitHub account).

For a couple of my stream temperature sites at Lake Tahoe – USGS-10336660 (39.10741, -120.1621) & USGS-10336645 (39.05185, -120.1185) – the erroneous values stuck out in calculations of air-water temperature metrics, which led me to download NOAA station data for Tahoe City (GHCND:USC00048758, black circles) to compare with the CONUS404 values at USGS-10336660 (red circles). Below you can see how the CONUS404 values hardly vary throughout an example year (WY2021) and an example decade (WY2011-2021).
![image](https://github.com/hytest-org/hytest/assets/30272838/861af0bc-c0d8-4146-9437-e1c100ab9868)
![image](https://github.com/hytest-org/hytest/assets/30272838/73c715f9-8d2d-4e84-a713-30debd6d3478)

...it also looks like the air temperature values for sites USGS-11455478 (38.09325, -121.7372) and USGS-11337190 (38.05214, -121.6888) are erroneous since nearby upstream sites (basically the same elevation) have more "normal" air temperature values. The upstream sites are USGS-11455420 (38.14904, -121.6889) and USGS-11313460 (38.05944, -121.5572), respectively. I didn't download any observed NOAA values for near these sites yet.
![image](https://github.com/hytest-org/hytest/assets/30272838/8a660fe1-7c89-47e8-927d-7d6b42ce88ef)
![image](https://github.com/hytest-org/hytest/assets/30272838/202cdb09-e47d-4b84-863c-8308d0c4f22d)

...I just noticed that all of the erroneous data for these sites are almost identical to each other. Could it be from an error in extracting the data by coordinates? [_AMELIA CONFIRMED THIS IS NOT THE ISSUE_] Otherwise, it seems to be some sort of systematic error in CONUS404 or my code [_I CONFIRMED MY CODE IS NOT THE ISSUE_]. Here are the erroneous air temperature values for three of these sites:
![image](https://github.com/hytest-org/hytest/assets/30272838/e7595b36-584d-4785-8c1d-ff69824702d4)

...here's a quick comparison with PRISM data at the USGS-10336660 site
![image](https://github.com/hytest-org/hytest/assets/30272838/764cca18-8ff2-45c0-909a-14970b9dbf94)

Thanks,

~Zach

**Comment by amsnyder** [comment]
For context, I believe the variable from CONUS404 that @zcjohnson3 is plotting is SKINTEMPMEAN from the wrfxtrm output files.

**Comment by amsnyder** [comment]
@gochis - do you have any thoughts on this? I know you and your team did the bias adjustment on the `T2` variable, so maybe you have familiarity with the temperature biases.

@rviger-usgs FYI

**Comment by zcjohnson3** [comment]
For some more context, here's a snapshot of a map showing how close some of the erroneous (small brown circles) can be to "normal" data sites (larger blue-green circles)
![image](https://github.com/hytest-org/hytest/assets/30272838/a93d1866-f022-4402-b258-cc9fea9b1e3b)

...or here's just air temperature metrics ("aampl_median" = air amplitude) for the same area. Notice how most of the circles are small (earlier phase) and green (larger amplitude), whereas the two erroneous sites are large (later phase) and blue-green (smaller amplitude).
![image](https://github.com/hytest-org/hytest/assets/30272838/5faafd03-39be-40c2-841a-82667e5523ac)

...or here's a site in Florida where the larger circles have identical values (phase = 213d, amplitude = 3.17degC) but all the other smaller circles have later timing (phase = ~195-199d) and larger amplitudes (~4.1-4.4degC)
![image](https://github.com/hytest-org/hytest/assets/30272838/79001e33-6640-4802-8d3b-0863e54132f1)

...or a couple of reservoir sites in South Carolina showing vastly different metrics
![image](https://github.com/hytest-org/hytest/assets/30272838/60b76015-213b-4fbc-b4a4-7585c4590313)

**Comment by zcjohnson3** [comment]
FYI, T2 data for the same site (USGS-10336660) looks good. Below compares "skin" (black), "T2" (red), and observed (blue, nearby NOAA station) for an example year.
![image](https://github.com/hytest-org/hytest/assets/30272838/b7bceb75-18f9-449a-97bb-8af6db8a5201)

**Comment by gochis** [comment]
Hi all...replying here via email...so not sure this is coming
through...assuming it is...

I'm not sure I really am following what the issue is that is being
discussed.  The last post by Zach seems to say that the air temperature
estimates from station and model align well but skin temperature does not.
That is very common as skin temperature is a surface radiative temperature
versus an air temperature...so I would say we would not expect great
agreement between air temperature and radiative temperature.  hope that
helps but feel free to chime in if there is more that needs to be dug into.
-Dave

**Reply by zcjohnson3** [reply]
Thanks @gochis. That was one of my original questions: is "skin" temperature air temperature or surface temperature? What I'm hearing is that I should be using the 2-m air temperature data ("T2").

**Reply by sfoks** [reply]
`T2` is air temperature at 2-meters above ground. From what I see online,` SKINTEMP `is calculated from a [surface energy balance](https://mailman.ucar.edu/pipermail/wrf-users/2012/002776.html).

I was trying to find more information/detail in the [WRF manual](https://www2.mmm.ucar.edu/wrf/users/wrf_users_guide/build/html/index.html) and in the [CONUS404 data release metadata](https://www.sciencebase.gov/catalog/item/6372cd09d34ed907bf6c6ab1), but we may need @gochis to further explain here if you want to know how skin temperature is calculated.

**Comment by gochis** [comment]
Hi all, yes, I commented on this above in the thread.  The SKINTEMP
variable is a radiative temperature from the land/soil/plant surface and it
is calculated as a residual in the energy and radiative exchange
component of NoahMP.  It should never be used as a surrogate for air
temperature for a host of reasons.
